apps/server/src/auth/auth.service.ts

```diff
@@ -4,6 +4,11 @@ import { JwtService } from '@nestjs/jwt';
 import { User } from '@prisma/client';
 import * as bcrypt from 'bcrypt';
 
+export interface JWT {
```

Comment:
@uptownhr pls update this to follow the JS coding convention:
all variables and methods should be camelCase.
all classes, interfaces, enums should be PascalCase.

ideally our ESLint should pick this up and we should have naming convention for the code as well as for file names. it would be great if you want to create a separate PR for this.

Reply:
updated